app/services/admin_service/applications_search.rb

```diff
@@ -22,6 +22,6 @@ def call
 private
 
   def default_scope
-    Application.joins(:school, :user).includes(:course, :lead_provider, :school, :user).order(id: :asc)
+    Application.left_joins(:school, :user).includes(:course, :lead_provider, :school, :user).order(id: :asc)
```

Comment:
Hmm - I think you just want to drop the school from the joins? From the code it doesn't look like they can search by school anyway?